web/src/app/shared/models/project.model.ts

```diff
@@ -1,3 +1,4 @@
+import { firestore } from 'firebase';
```

Comment:
Should these be group separately, the same as other places?

Reply:
Done